packages/base/Grid/src/GridItem/utils/get-class-names-for-breakpoint.ts

```diff
@@ -0,0 +1,218 @@
+import type { BreakpointKeys } from '@toptal/picasso-provider'
+
+import type { GridSize } from '../../types'
+
+const growMapping: { [K in BreakpointKeys]: { [K in string]: string } } = {
+  xs: {
+    grow: 'xs:grow',
+    'grow-0': 'xs:grow-0',
+  },
+  sm: {
+    grow: 'sm:grow',
+    'grow-0': 'sm:grow-0',
+  },
+  md: {
+    grow: 'md:grow',
+    'grow-0': 'md:grow-0',
+  },
+  lg: {
+    grow: 'lg:grow',
+    'grow-0': 'lg:grow-0',
+  },
+  xl: {
+    grow: 'xl:grow',
+    'grow-0': 'xl:grow-0',
+  },
+}
+
+const maxWidthMapping: { [K in BreakpointKeys]: { [K in string]: string } } = {
```

Comment:
NIT: I find using `Record` a bit more readable than using the indexed objects syntax directly, no need to change if you don't like it
```suggestion
const growMapping:  Record<BreakpointKeys, Record<string, string>> = {
  xs: {
    grow: 'xs:grow',
    'grow-0': 'xs:grow-0',
  },
  sm: {
    grow: 'sm:grow',
    'grow-0': 'sm:grow-0',
  },
  md: {
    grow: 'md:grow',
    'grow-0': 'md:grow-0',
  },
  lg: {
    grow: 'lg:grow',
    'grow-0': 'lg:grow-0',
  },
  xl: {
    grow: 'xl:grow',
    'grow-0': 'xl:grow-0',
  },
}

const maxWidthMapping: Record<BreakpointKeys, Record<string, string>> = {
```